src/test/resources/File/AtomicWrite.pm

```diff
@@ -0,0 +1,79 @@
+package File::AtomicWrite;
+
+use strict;
+use warnings;
+
+use version;
+
+# Always recent enough
+our $VERSION = 10000.0.0;
```

Comment:
I think we should declare here the API version we are mocking. Then if a later version changes, we get a failure and get reminded to go and check we updated the mock to match the behaviour of the real thing?